### PR TITLE
feat(ejb): add deprecation notice in release note

### PR DESCRIPTION
**Pull request by baptistemesta**
Relates to [BS-18](https://bonitasoft.atlassian.net/browse/BR-18)

### CODE REVIEW

File: md/release-notes.md [hunk 1]
```diff
@@ -39,6 +39,12 @@ In order to be compatible with Java 11, you must update the following plug-ins d
 ### New embedded AngularJS filter to resolve business object lazy references
 More info on how to use it [here](variables.md).
 
+## Deprecation
+
+### EJB
+
+EJB communication protocol with the engine is now deprecated, Removal is planned for 7.10 versiion
```
Comment (educhastenier) [inline md/release-notes.md]:
```suggestion
EJB communication protocol with the engine is now deprecated. Removal is planned for 7.10 version.
```

**Comment by educhastenier** [comment]
if target removal 7.10 is confirmed...

**Comment by tnguyen1** [comment]
retest this please

**Comment by tnguyen1** [comment]
retest this please

**Comment by tnguyen1** [comment]
retest this please

**Comment by tnguyen1** [comment]
retest this please

**Pull request merged**